Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -190,9 +190,6 @@ class DataPipelineImplementation: DataPipelineInstance {
         // Consolidate all Apple platforms under iOS
         deviceAttributesProvider.getDefaultDeviceAttributes { defaultDeviceAttributes in
             let deviceAttributes: [String: Any] = defaultDeviceAttributes
-                .mergeWith([
-                    "last_used": self.dateUtil.now
-                ])
```

Comment:
Migration question: Do we also need to remove `last_used` from the unprocessed tasks we get from background queue during migration ? We do get this property in certain json payloads and we send them as is to CDP.

Reply:
If yes, then we can make the change in another PR, no need to hold this PR.

Reply:
Yup, we will need to remove it, along with that check the `timestamp` in general.